src/main/java/me/botsko/prism/commands/WandCommand.java

```diff
@@ -82,160 +87,137 @@ public void handle(CallInfo call) {
             }
         }
 
-        Material item_material = null;
+        Material itemMaterial = null;
 
         if (toolKey != null) {
-            item_material = Material.matchMaterial(toolKey);
+            itemMaterial = Material.matchMaterial(toolKey);
         }
 
         String wandOn = "";
-        String item_name = "";
+        String itemName = "";
         StringBuilder parameters = new StringBuilder();
-        if (item_material != null) {
-            item_name = Prism.getItems().getAlias(item_material, null);
-            wandOn += " on a " + item_name;
+        if (itemMaterial != null) {
+            itemName = Prism.getItems().getAlias(itemMaterial, null);
+            wandOn += " on a " + itemName;
         }
 
         for (int i = (isInspect ? 1 : 2); i < call.getArgs().length; i++) {
-            if (parameters.length() == 0) {
-                parameters.append(" using:" + ChatColor.GRAY);
-            }
             parameters.append(" ").append(call.getArg(i));
         }
 
-        if (ItemUtils.isBadWand(item_material)) {
-            call.getPlayer().sendMessage(
-                    Prism.messenger.playerError("Sorry, but you may not use " + item_name + " for a wand."));
+        if (ItemUtils.isBadWand(itemMaterial)) {
+            final String itemNameFinal = itemName;
+            Prism.messenger.sendMessage(call.getPlayer(),
+                    Prism.messenger.playerError(Il8nHelper.getMessage("wand-bad")
+                            .replaceText(Pattern.compile("<itemName>"),
+                                    builder -> Component.text().content(itemNameFinal))));
```

Comment:
Codacy found an issue: ['lambda arguments' has incorrect indentation level 36, expected level should be one of the following: 32, 34.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6279951)